mob

```diff
@@ -240,23 +241,10 @@ if minor_ver > 0:
     parent_tag = os.path.join(repo, mobconf['image']['target'] + ':' + major_ver + "_" + str(minor_ver - 1))
 
 if args.action == "default-tag":
-    print(default_tag, end = '')
+    print(default_tag, end='')
     sys.exit(0)
 
-###
-# Calculate command
-###
-
-build_env = [
-            "RUST_BACKTRACE=full",
-            "SGX_MODE=HW" if args.hw else "SGX_MODE=SW",
-            "IAS_MODE=PROD" if args.ias_prod else "IAS_MODE=DEV",
-            ]
-
-# Calculate bash commands
-if args.action == "prompt" or args.action == "image":
-    bash_commands = None
-else:
+if not (args.action == "prompt" or args.action == "image"):
```

Comment:
`if args.actions not in ['prompt', 'mage']:`

Reply:
After investigating more, I believe this condition is unreachable anyway, so I removed it.

Reply:
(because of argparse: `parser.add_argument("action", choices=["prompt", "image", "default-tag"], ...`)